physical/s3/s3.go

```diff
@@ -188,16 +188,17 @@ func (s *S3Backend) Get(ctx context.Context, key string) (*physical.Entry, error
 	if resp == nil {
 		return nil, fmt.Errorf("got nil response from S3 but no error")
 	}
+	defer resp.Body.Close()
 
-	data := make([]byte, *resp.ContentLength)
-	_, err = io.ReadFull(resp.Body, data)
+	data := bytes.NewBuffer(nil)
+	_, err = io.Copy(data, bytes.NewBuffer(resp.Body))
```

Comment:
I believe this should just be resp.Body w/o the bytes.NewBuffer()

Reply:
Oh yeah good point, I did that before looking up whether it was a byte slice or reader and forgot to go back and change it.